letsencrypt/client/auth_handler.py

```diff
@@ -201,9 +203,7 @@ def _assign_responses(self, flat_list, ichall_dict):
 
     def _path_satisfied(self, dom):
         """Returns whether a path has been completely satisfied."""
-        return all(
-            None != self.responses[dom][i] and "null" != self.responses[dom][i]
-            for i in self.paths[dom])
+        return all(self.responses[dom][i] is not None for i in self.paths[dom])
```

Comment:
This is what I thought... The current master does not check for ``False`` either, which I suggested as a bug some time ago on IRC. To keep the PR in scope I would leave it as is, and submit a separate (simultaneous?) PR to fix it. Okay?